pkg/apis/topology/v1alpha1/generated.proto

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This package imports things required by build scripts, to force `go mod` to see them as dependencies
+
+// This file was autogenerated by go-to-protobuf. Do not edit it manually!
+
+syntax = "proto2";
+
+package v1alpha1;
+
+import "k8s.io/apimachinery/pkg/util/intstr/generated.proto";
+
+// Package-wide variables from generator "generated".
+option go_package = "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha1";
+
+// +protobuf=true
+message AttributeInfo {
+  optional string name = 1;
+
+  optional string value = 2;
+}
+
+// +protobuf=true
+message CostInfo {
+  optional string name = 1;
+
+  optional int64 value = 2;
```

Comment:
I wander how this could be generated as int64, when original CostInfo.value is just int?

Reply:
Hmm, there is no such type as `int` in protobuf 🤔  We should change CostInfo.value to `int64` or `int32`. What do you think, do this in a separate PR or a separate patch in this PR? @AlexeyPerevalov @swatisehgal (?)